.github/workflows/main.yml

```diff
@@ -5,11 +5,22 @@ on: [push, pull_request]
 jobs:
   test:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - "lint"
+          - "test-centos:6"
+          - "test-centos:7"
+          - "test3-centos:stream8"
+          - "test3-centos:stream9"
     steps:
       - uses: actions/checkout@v2.4.0
       - name: Set up Python
         uses: actions/setup-python@v2.3.1
+        if: matrix.target == 'lint'
```

Comment:
Wouldn't it be cleaner to write a separate lint job for this? So
```yaml
jobs:
  lint:
    # ...

  test:
    # ...
```

Reply:
It would, I was just avoiding duplication if the other steps